pandas/core/arrays/datetimelike.py

```diff
@@ -2514,4 +2515,8 @@ def dtype_to_unit(dtype: DatetimeTZDtype | np.dtype) -> str:
     """
     if isinstance(dtype, DatetimeTZDtype):
         return dtype.unit
+    elif isinstance(dtype, ArrowDtype):
+        if dtype.kind not in "mM":
+            raise ValueError(f"{dtype=} does not have a resolution.")
```

Comment:
typo here?

Reply:
I don't think so? (If you're referring to `dtype=`, it was intentional to render e.g. `"dtype=int64[pyarrow] does not have a resolution"`)

Reply:
maybe im misunderstanding how f-strings work.  id expect this to render "int64[pyarrow]= does not have a resolution"

Reply:
```python
In [2]: import pandas as pd

In [3]: import pyarrow as pa

In [4]: dtype = pd.ArrowDtype(pa.int64())

In [5]: f"{dtype=} does not have a resolution"
Out[5]: 'dtype=int64[pyarrow] does not have a resolution'
```

Reply:
neat!